luomu-common/src/macaddr.rs

```diff
@@ -38,11 +72,27 @@ impl std::ops::Deref for MacAddr {
 #[cfg(test)]
 mod tests {
     use super::MacAddr;
+    use std::convert::TryFrom;
 
     #[test]
     fn test_fmt_debug() {
         let mac: MacAddr = [0x00, 0xff, 0x11, 0xee, 0x22, 0xdd].into();
         let debug = format!("{:?}", mac);
         assert_eq!(&debug, "00:ff:11:ee:22:dd");
     }
+
+    #[test]
+    fn test_try_from_u64() {
+        let i0 = 0x0000000000000000;
+        assert!(MacAddr::try_from(i0).is_ok());
+
+        let i1 = 0x0000FFFFFFFFFFFF;
+        assert!(MacAddr::try_from(i1).is_ok());
+
+        let i2 = 0x0001000000000000;
+        assert!(MacAddr::try_from(i2).is_err());
+
+        let i3 = 0xFFFFFFFFFFFFFFFF;
+        assert!(MacAddr::try_from(i3).is_err());
+    }
```

Comment:
Could you add a test for valid case as well. Just to make sure the byte order is right in the end result. For my experience I can tell that it is easy to get wrong when refactoring and having a test which catches that would be golden.